### PR TITLE
Create 2015-06-17-shape-lines

### PR DESCRIPTION
**Pull request by aneda**
@chriddyp  is this too long for the "simple" section?

### CODE REVIEW

File: _posts/python/shapes/2015-06-17-shape-lines [hunk 1]
```diff
@@ -0,0 +1,83 @@
+---
+name: Lines
```
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shape-lines]: Update to "Vertical and Horizontal Lines"

File: _posts/python/shapes/2015-06-17-shape-lines [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    },
+    
+    'width' : 600,
+    'height' : 600,
+    
+    'shapes':[
+
+        # Line Vertical
+        {
+        'type' : 'line',
```
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shape-lines]: Indent this block

File: _posts/python/shapes/2015-06-17-shape-path [hunk 3]
```diff
@@ -0,0 +1,82 @@
+---
+name: Basic Path
```
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shape-path]: update to: "Basic SVG Paths"

File: _posts/python/shapes/2015-06-17-shape-path [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+plot_url: https://plot.ly/~neda/4117
+language: python
+suite: shape
+order: 4
+---
```
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shape-path]: let's add a quick description field here:

```
description: "path" shapes are a wrapper around SVG's <path> element that allow for data or paper coordinates. <a href="https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Paths">Learn more about the "path" syntax.</a>
```

File: _posts/python/shapes/2015-06-17-shape-path [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+        'showgrid' : False,
+    },
+    
+    'width' : 600,
+    'height' : 600,
+    
+    'shapes':[
+    # Quadratic Bezier Curves
+    {
+    'type' : 'path',
```
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shape-path]: indent here

File: _posts/python/shapes/2015-06-17-shape-path [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+    'yaxis':{
+        'range':[0, 11],
+        'showgrid' : False,
+    },
+    
+    'width' : 600,
+    'height' : 600,
+    
+    'shapes':[
+    # Quadratic Bezier Curves
```
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shape-path]: indent here

File: _posts/python/shapes/2015-06-17-shapes-circle [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+    },
+    
+    'width' : 600,
+    'height' : 600,
+    
+    'shapes':[
+
+        # unfilled circle
+        {
+         'type': 'circle',
```
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shapes-circle]: indent here

File: _posts/python/shapes/2015-06-17-shapes-circle [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+    
+    'yaxis':{
+        'range':[0, 4.5]
+    },
+    
+    'width' : 600,
+    'height' : 600,
+    
+    'shapes':[
+
```
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shapes-circle]: rm this space

File: _posts/python/shapes/2015-06-17-shape-lines [hunk 9]
```diff
@@ -0,0 +1,83 @@
+---
+name: Lines
+plot_url: https://plot.ly/~neda/4020
+language: python
+suite: shape
```
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shape-lines]: you'll need to add a few things to get this to show up:
- an "index" file inside that footer that looks just like this: https://github.com/plotly/documentation/blob/gh-pages/_posts/python/log/2015-04-09-log_python_index.html except replace all of the "suite": "log" with "shape"
- a thumbnail in here: https://github.com/plotly/documentation/tree/gh-pages/all_static/images

File: _posts/python/shapes/2015-06-17-shape-path (hunk 5 above)
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shape-path]: if you're using sublime text, could you install the PEP8 validator (https://github.com/dreadatour/Flake8Lint) and run these through a syntax validation? that'll catch things like indents and extra spaces etc. let me know if you need any help!

File: _posts/python/shapes/2015-06-17-shape-lines [hunk 10]
```diff
@@ -0,0 +1,83 @@
+---
+name: Vertical and Horizontal Lines Positioned Relative to the Plot
```
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shape-lines]: woops, just realized that the default is `"xref": "x"`, so this should be
"Vertical and Horizontal Lines Positioned Relative to the Axes"

File: _posts/python/shapes/2015-06-17-shapes-rect [hunk 11]
```diff
@@ -0,0 +1,67 @@
+---
+name: Rectangle
```
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shapes-rect]: maybe:
"name: Rectangle Positioned Relative to the Axes"

**Comment by chriddyp** [comment]
@aneda these are great! just those comments above. in another PR, we should add an example of lines and rectangles with "xref" and "yref" positioned relative to "x" and "y"

**Comment by aneda** [comment]
@chriddyp  is this good for line "xref"/ "yref":  https://plot.ly/~neda/4196

**Comment by chriddyp** [comment]
looks good!

**Pull request merged**